tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain.md

```diff
@@ -0,0 +1,110 @@
+---
+title: From Relay Chain to Parachain
+description: Learn how to perform a reserve-backed asset transfer between a relay chain and a parachain using XCM for cross-chain interoperability.
+---
+
+# From Relay Chain to Parachain
+
+## Introduction
+
+[Cross-Consensus Messaging (XCM)](/develop/interoperability/intro-to-xcm/){target=\_blank} facilitates asset transfers both within the same consensus system and between different ones, such as between a relay chain and its parachains. For cross-system transfers, two main methods are available:
+
+- [**Asset teleportation**](https://paritytech.github.io/xcm-docs/journey/transfers/teleports.html){target=\_blank} - a simple and efficient method involving only the source and destination chains, ideal for systems with a high level of trust
+- [**Reserve-Backed transfers**](https://paritytech.github.io/xcm-docs/journey/transfers/reserve.html){target=\_blank} - involves a trusted reserve holding real assets and mints derivative tokens to track ownership. This method is suited for systems with lower trust levels
+
+In this tutorial, you will learn how to perform a reserve-backed transfer of DOT between a relay chain (Polkadot) and a parachain (Astar).
+
+## Prerequisites
+
+To send messages between different consensus systems, you must first open HRMP channels. For detailed guidance, refer to the [XCM Channels](/develop/interoperability/xcm-channels/#xcm-channels){target=\_blank} article before for further information about.
```

Comment:
If the focus of this example will be on specific chains, here maybe just say "When adapting this tutorial for other chains, before you can send messages between different consensus systems, you must first open HRMP channels." Or something

Reply:
Idk that can probably be cleaned up a bit more to be more concise